extras/device_client/lib/view/home_page.dart

```diff
@@ -7,6 +7,7 @@ import 'package:device_client/view/current_data_page.dart';
 import 'package:device_client/view/files_page.dart';
 import 'package:flutter/material.dart';
 import 'package:provider/provider.dart';
+// import 'package:web/web.dart' as html;
```

Comment:
If you are going to commit commented-out code, add a TODO: with an explanation of what you plan to do with it.